cmd/commander/commander.go

```diff
@@ -53,6 +53,15 @@ Tests are always executed in alphabetical order.
 
 Examples:
 
+Directory test:
+commander test --dir /your/dir/
+
+Stdin test:
+cat commander.yaml | commander test -
+
+HTTP test:
+test https://your-url/commander_test.yaml
```

Comment:
A little bit picky, but you wrote `commander test` the first two examples, in the third you only wrote `test`

Reply:
`cat commander.yaml | commander test - ` still uses `commander`. 
Just add commander to the other two commands :p